api/campusdish_interface.py

```diff
@@ -36,39 +36,65 @@ def get_schedule_data(restaurant: str) -> dict:
     schedule time use int because frontend work with int
     schedule time is (100*hours)+minutes, where hours is in 24-hour time
     '''
-    url = 'https://uci.campusdish.com/LocationsAndMenus/'
-    if restaurant == 'Anteatery':
-        url += 'TheAnteatery'
-    else:
-        url += restaurant
-    schedule = {}
-    soup = bs(requests.get(url).text, 'html.parser')
-    meal_period = soup.select('.mealPeriod')[:5]
-    location_times = soup.select('span[class=location__times]')[:5]
-    for idx, meal in enumerate(meal_period):
-        meal = meal.getText().lower()
-        times = location_times[idx].getText().split(' - ')
-        if re.match(r"^\d?\d:\d\d(AM|PM)$", times[0]) and re.match(r"^\d?\d:\d\d(AM|PM)$", times[1]):
-            start = normalize_time_from_str(times[0])
-            end = normalize_time_from_str(times[1])
-            schedule[meal] = {"start": start, "end": end}
+    try:
```

Comment:
The diff here looks big but all this does is put the existing code inside this try block without modifying it.

Reply:
Ok, I will create a pull request later as well to make changes to these codes so that it gets the schedule data correctly.